kernel/Kconfig.device

```diff
@@ -45,6 +45,14 @@ config KERNEL_INIT_PRIORITY_OBJECTS
 	  priority needs to be higher than minimal default initialization
 	  priority.
 
+config KERNEL_INIT_PRIORITY_LIBC
+	int "LIBC initialization priority"
+	default 30
```

Comment:
I think malloc should initialize after kernel objects as malloc initialization *could* involve initializing various kernel objects like mutexes. Therefore, I think this value should be between KERNEL_INIT_PRIORITY_OBJECTS and KERNEL_INIT_PRIORITY_DEFAULT.

Reply:
have updated as your comment, pls review again